### ISSUE TITLE
Add new wallets to Documentation

### ISSUE THREAD

**Issue by franculio** [Giveth/giveth-planning#1647]
Going through the documentation I noticed that we could update the new wallets added in the last version of the Dapp released. 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMTU4QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ab4728cd14ed1876363fc925f680342c4eb54143/Captura%20de%20pantalla%202023-10-27%20a%20las%2017.15.58.png)

@divine-comedian could you confirm which wallets need to be added?

**Comment by divine-comedian** [comment]
Yeah well I think this will also be changing soon the UX for connecting wallets 
https://github.com/Giveth/giveth-dapps-v2/issues/3370

and will also change again when we add Solana to the dapp perhaps in about a month and a half

**Comment by oyealmond** [comment]
Hello @divine-comedian, I will move this issue to Notion. Is there something else we should consider to update documentation about wallets? Or this is it.

**Comment by ahmadabugosh** [comment]
@oyealmond I would add
1) Now that Solana is live, we should make it clear that those are counted as two different wallets atm (and they aren't connected) 
2) Also mention the new chains we recently added like Polygon, Arbitrum  etc.
